sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -378,6 +378,15 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val PARQUET_FILTER_PUSHDOWN_TIMESTAMP_ENABLED =
+    buildConf("spark.sql.parquet.filterPushdown.timestamp")
+      .doc("If true, enables Parquet filter push-down optimization for Timestamp. " +
+        "This configuration only has an effect when 'spark.sql.parquet.filterPushdown' is " +
+        "enabled and Timestamp stored as TIMESTAMP_MICROS or TIMESTAMP_MILLIS type.")
```

Comment:
Shell we note `INT64` here?

Reply:
I think end users have a better understanding of `TIMESTAMP_MICROS` and `TIMESTAMP_MILLIS`.

Reply:
... I don't think users will understand any of them ..

Reply:
You need to explain how to use `spark.sql.parquet.outputTimestampType` to control the Parquet timestamp type  Spark uses to writes parquet files.

Reply:
I would just note that push-down doesn't work for INT96 timestamps in the file. It should work for the others.